librz/il/rzil_export.c

```diff
@@ -329,15 +329,15 @@ static void il_opdmp_append(RzILOp *op, RzStrBuf *sb, PJ *pj) {
 static void il_opdmp_load(RzILOp *op, RzStrBuf *sb, PJ *pj) {
 	RzILOpLoad *opx = op->op.load;
 	if (sb) {
-		rz_strbuf_append(sb, "load(key:");
+		rz_strbuf_appendf(sb, "load(mem:%u, key:", (unsigned int)opx->mem);
```

Comment:
maybe let's avoid `unsigned int` and instead use `ut32`

Reply:
`%u` is `unsigned int` so I **must** cast it to that when it's `ut32`.

Reply:
> `%u` is `unsigned int` so I **must** cast it to that when it's `ut32`.

not really. ut32 is unsigned int